examples/autoscaler-webhook/README.md

```diff
@@ -0,0 +1,10 @@
+# Simple Webhook Autoscaler Service
+
+This service provides an example of using the [Agones API](https://godoc.org/agones.dev/agones/pkg/client/clientset/versioned/typed/stable/v1alpha1) to produce the number of GameServers in a Fleet.
+
+## Autoscaler Service
+The service exposes an endpoint which allows client calls to custom scaling logic.
+
+When the endpoint is called and a target Replica count is calculated and scale flag. It returns the JSON encoded FleetAutoscaleReview.
+
+To learn how to deploy the fleet to GKE, please see the tutorial [Create a Fleet (Go)](../../docs/create_fleet.md).
```

Comment:
I would like more details of that autoscaler implementation with some example on how it behave.

Reply:
I added the autoscaling flow example with some calculations into this file.